sympy/printing/tests/test_latex.py

```diff
@@ -3156,3 +3156,7 @@ def test_Array():
 def test_latex_with_unevaluated():
     with evaluate(False):
         assert latex(a * a) == r"a a"
+
+
+def test_latex_disable_split_super_sub():
+    assert latex(Symbol('u^a_b'), disable_split_super_sub=True) == 'u\\^a\\_b'
```

Comment:
It would be good in these tests to show also what happens when the option is not used.